### PR TITLE
Feature/division recognition config

### PR DESCRIPTION
**Pull request by anovikov93**

### CODE REVIEW

File: Core/Tenders.Core/Services/DivisionRecognitionConfigService.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    using Newtonsoft.Json;
+    using Tenders.Core.Abstraction.Services;
+    using Tenders.Core.Models;
+
+    /// <inheritdoc/>
+    public class DivisionRecognitionConfigService : IDivisionRecognitionConfigService
+    {
+#pragma warning disable S2933 // Fields that are only assigned in the constructor should be "readonly"
+#pragma warning disable IDE0044 // Add readonly modifier
+        private string divisionsUnit = string.Empty;
```
Comment (Anny777) [inline Core/Tenders.Core/Services/DivisionRecognitionConfigService.cs]: Field need be readonly

File: Core/Tenders.Core/Services/DivisionRecognitionConfigService.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+namespace Tenders.Core.Services
+{
+    using System.Web;
+    using Newtonsoft.Json;
+    using Tenders.Core.Abstraction.Services;
+    using Tenders.Core.Models;
+
+    /// <inheritdoc/>
+    public class DivisionRecognitionConfigService : IDivisionRecognitionConfigService
+    {
```
Comment (Anny777) [inline Core/Tenders.Core/Services/DivisionRecognitionConfigService.cs]: There must be a constructor. In parameters config builder and config provider. See other config services.

File: Core/Tenders.Core/Abstraction/Services/IDivisionRecognitionConfigService.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+// </copyright>
+
+namespace Tenders.Core.Abstraction.Services
+{
+    using Tenders.Core.Models;
+
+    /// <summary>
+    /// Division recognition config service.
+    /// </summary>
+    public interface IDivisionRecognitionConfigService
```
Comment (tatianaIljina) [inline Core/Tenders.Core/Abstraction/Services/IDivisionRecognitionConfigService.cs]: Is it necessary?

File: Core/Tenders.Core/Models/DivisionUnitContainer.cs [hunk 4]
```diff
@@ -0,0 +1,17 @@
+﻿// <copyright file="DivisionUnitContainer.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace Tenders.Core.Models
+{
+    /// <summary>
+    /// Container for division unit.
+    /// </summary>
+    public class DivisionUnitContainer
```
Comment (tatianaIljina) [inline Core/Tenders.Core/Models/DivisionUnitContainer.cs]: Is it necessary?

Comment (Anny777) [reply Core/Tenders.Core/Models/DivisionUnitContainer.cs]: don't think

**Pull request merged**